psi-ms.obo

```diff
@@ -22663,6 +22663,18 @@ relationship: has_value_type xsd\:float ! The allowed value-type for this CV ter
 relationship: has_value_concept NCIT:C25330 ! Duration
 relationship: has_units UO:0000010 ! second
 
+id: MS:4000068
```

Comment:
Requires `[Term]` stanza opening syntax. I didn't see it before.

Reply:
great, thanks for fixing!